chapters/synchronous.tex

```diff
@@ -127,30 +122,32 @@ \subsection{Rationale for Clocked Semantics}\label{rationale-for-clocked-semanti
     controller part, and different integer multiples are given, then the
     translator has to accept this (no error is detected).
   \end{enumerate}
+  Note: Clocked systems can mix different sampling rates
+  in well-defined ways when needed.
 \item
-  Due to the automatic sample and hold semantics, all variables
-  assigned in a when-clause of the above kind must have an initial value
+  Fewer initial conditions are needed, as only a subset of clocked
+  variables need initial conditions - the clocked state variables \cref{clocked-state-variables}.
```

Comment:
The reference to _clocked-state-variables_ doesn't look quite integrated into the sentence, perhaps use a parenthesis again?
```suggestion
  variables need initial conditions -- the clocked state variables (see \cref{clocked-state-variables}).
```

Reply:
Ok